common/source/docs/common-electro-permanent-magnet-V3.rst

```diff
@@ -57,7 +57,7 @@ Firmware setup
 
```

Comment:
I suggest removing this entire section....UAVCAN is supported now....3.4 is old and doesnt need mentioning....